src/sentry/plugins/providers/repository.py

```diff
@@ -83,3 +83,7 @@ def delete_repository(self, repo, actor=None):
 
     def compare_commits(self, repo, start_sha, end_sha, actor=None):
         raise NotImplementedError
+
+    @staticmethod
+    def ignore_commit(message):
+        return 'SKIP-SENTRY' in message
```

Comment:
I would also suggest naming this `should_ignore_commit` to suggest that it just returns a boolean. Otherwise, it sounds like it's actually doing an action.